codeforboston/maple
# Migrating PasswordInput component to Bootstrap

# Summary

This PR migrates the PasswordInput component to Bootstrap: #1460 

# Checklist

- [na] On the frontend, I've made my strings translate-able.

# Screenshots

N/A (Looks identical to production)

# Known issues

N/A

# Steps to test/reproduce

1. Go to any page while signed out and click Sign In from the Nav
1. See that the PasswordInput displays as expected
